docs/OESDLBugBar.md

```diff
@@ -0,0 +1,84 @@
+
+Enclave security boundary is quite interesting, everything that is outside this boundary is adversarial which can fully control the existence of the enclave but can’t access or modify the data over inside the enclave crossing the enclave security boundary. Things that are outside the enclave security boundary are the current application, other enclaves, OS kernel, VMM/Hypervisor & SMM. Except the CPU everything is untrusted for enclaves. 
+
+
+## Enclave – Vulnerability Type Pivot 
+-------------------------------------
+
+* ### Spoofing
+	**Critical**
+    * A malicious enclave spoofs the identity of a victim enclave
+  	
+    -------------
+
+* ### Tampering	
+    **Critical**
+    * Any modification of enclave image/data that compromise the integrity of the enclave
+
+            Examples:
+            •	Tampering the enclave image in plain text and keep the measurements same
+    
+    -------------
+
+* ### Repudiation
+    * N/A
+
+    -------------
+
+* ### Information Disclosure
+    **Critical**
+	* Disclosing enclave memory contents in the plain-text (not encrypted) 
+
+            Examples:
+            •	OCALLS returning uninitialized memory from enclave to outside	
+
+    **Important**
+    * Any cases where the attacker can bypass the enclave security boundary to read information belonging to an enclave on the system that was not intended or designed to be exposed
+
+            Examples:
+            •	Unintentional read access to memory contents in enclave space from outside (application, OS kernel, Hypervisor)
+            •	OCALLS leaking more information out of enclave	
+
+    -----------
+
+* ### Denial of Service
+	* N/A
+
+            Enclave threat model doesn’t protect against DoS. Since the enclave execution is under the adversarial control, which is the host OS, the threat model assumes the host OS to be untrusted, so DoS attacks are unprotected.
+
+    -----------
+
+* ### Elevation of Privilege	
+    **Critical**
+    * Any cases where an attacker can bypass the security boundary and execute arbitrary code within an enclave context
+
+            Examples:
+            •	Attacker able to invoke any executable code other than enclave entry points
+
+    * All exploitable write AVs (Access Violations), integer overflows and other crashes
+
+            Examples:
+            •	Any memory corruption bug within enclave that can be triggered from an attacker outside of the enclave	
+        
+    **Important**
+	* Access to untrusted memory in an unsafe manner
+
+            Examples:
+            •	Any untrusted memory that is accessed multiple times without (multiple fetch) copying it to enclave memory.  
```

Comment:
nit: (multiple fetch) shoud go before the word "without"

Reply:
will fix this